### ISSUE TITLE
each button in image editor will trigger form submit

### ISSUE THREAD

**Issue by sanluan** [scaleflex/filerobot-image-editor#484]
if we add the plugin to a form , then our form is sent if we click on one of the plugin's buttons.
```
<form onsubmit="alert('submit')">
<div id="editor_container"></div>
</form>
<script>
var filerobotImageEditor = new FilerobotImageEditor(
  document.querySelector("#editor_container"),
  filerobotImageEditorConfig,
);

filerobotImageEditor.render({
  onClose: (closingReason) => {
    filerobotImageEditor.terminate();
  },
});
</script>
```

**Comment by sanluan** [comment]
Temporary workaround:
```
$("#editor_container").on("click","button",function(){
    return false;
});
```

**Comment by AhmeeedMostafa** [comment]
@sanluan this will be solved in the new major release

**Comment by AhmeeedMostafa** [comment]
@sanluan after checking the issue was related to @scaleflex/ui package, it is updated now in the released minor version (4.9.0) please recheck and if there is any comment reopen the issue.
